config/samples/resources/edgenetworknetwork/edgenetwork_v1beta1_edgenetworknetwork.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: edgenetwork.cnrm.cloud.google.com/v1beta1
+kind: EdgenetworkNetwork
+metadata:
+  name: edgenetworknetwork-sample
+spec:
+  description: "A sample edgenetwork network"
+  location: sample-location
```

Comment:
Could you use real values for location and zone?

Reply:
Cloud locations are well-known, but edge zone names are private. I've replaced these with a real location and a fake (but realistic) zone name.